extensions/adobe/experience/customerJourneyManagement/message-tracking-link.schema.json

```diff
@@ -0,0 +1,72 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/experience/customerJourneyManagement/message-tracking-link",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Adobe CJM ExperienceEvent - Tracking link details",
+  "description": "Describes fields associated with the tracking link through which interaction is made by the end user.",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+
+  "definitions": {
+    "messageTrackingLink": {
+      "properties": {
+        "xdm:interactionType": {
+          "title": "Interaction Type",
+          "type": "string",
+          "description": "Specify action made by user on a message.",
+          "enum": ["click", "open", "action"],
+          "meta:enum": {
+            "click": "Link in a message is clicked by the user.",
+            "open": "Either Message is open by the user, or in case of push, clicking the message opened the mobile application.",
+            "action": "Custom action taken on push notifcation, like clicking on some button in push notification."
+          }
+        },
+        "@id": {
+          "title": "Tracking URL ID",
+          "type": "string",
+          "description": "Unique Identifier of the Link"
+        },
+        "xdm:description": {
```

Comment:
I am not sure there will be any tracking URL description. Only URL label is needed fro reporting.